README.md

```diff
@@ -5,6 +5,13 @@ Taskswamp is a scriptlet that creates a `tmux` session with multiple `task`
 reports and allows those reports to be easily refreshed.  Each tmux window
 shows a single taskwarrior report.
 
+As of 2021/05/28, upstream taskwarrior can corrupt your data if called simultaneously.
+Taskswamp is already fixing this issue as part of https://github.com/taskswamp/taskswamp/commit/2f42a34deb0e09f37483bfa38ed56ee058597422 .
+For full compatibility with Taskswamp, you should wrap your other individual task
+to include the same locking mecanism. For example
+
+    alias task='flock ~/.task/.taskswamp.lock task'
```

Comment:
I think that's not the best place for the added text.  It certainly doesn't fit under the `What is this?` heading, nor before `Quick start`.

As a rule, documentation should state the common case first and exceptions afterwards.  So, the `Quick start` section could, for instance, say something along the lines of —

> If you run `task` concurrently and your Taskwarrior version is $foo, see the $bar section

— where the mentioned section would explain the problem and the solution.  TIMTOWTDI, of course; this is just an idea.